bigtable/google/cloud/bigtable/table.py

```diff
@@ -373,3 +406,50 @@ def _create_row_request(table_name, row_key=None, start_key=None, end_key=None,
         message.rows.row_ranges.add(**range_kwargs)
 
     return message
+
+
+def _mutate_rows_request(table_name, rows):
+    """Creates a request to read rows in a table.
+
+    :type table_name: str
+    :param table_name: The name of the table to read from.
+
+    :type rows: list
+    :param rows: List or other iterable of :class:`.DirectRow` instances.
+    
+    :rtype: :class:`data_messages_v2_pb2.MutateRowsRequest`
+    :returns: The ``MutateRowsRequest`` protobuf corresponding to the inputs.
+    :raises: :class:`TooManyMutationsError <exceptions.TooManyMutationsError>` 
+             if the number of mutations is grater than 100.000
+    """
+    request_pb = data_messages_v2_pb2.MutateRowsRequest(table_name=table_name)
+    mutations_count = 0
+    for row in rows:
+        entry = request_pb.entries.add()
+        entry.row_key = row.row_key
+        for mutation in row._get_mutations(None):
+            mutations_count += 1
+            entry.mutations.add().CopyFrom(mutation)
+    if mutations_count > 100000:
+        raise TooManyMutationsError('Maximum number of the entries mutations '
+                                    'is 100000')
+    return request_pb
+
+
+def _check_rows(table_name, rows):
+    """Checks that all rows belong to the table.
+
+    :type table_name: str
+    :param table_name: The name of the table to read from.
+
+    :type rows: list
+    :param rows: List or other iterable of :class:`.DirectRow` instances.
+    """
+    for row in rows:
+        if not isinstance(row, DirectRow):
+            raise TypeError('Bulk processing can not be applied for '
+                            'conditional or append mutations.')
+        if row.table.name != table_name:
+            raise RowBelongingError(
```

Comment:
I would prefer to inherit from ValueError, if it is OK in a scope of the project.